src/qibolab/channel.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class NamedChannel:
```

Comment:
Do we need unnamed channels at all?

I.e. maybe the idea of the higher layer here is even fine (I should think more about it), but to me the focus should not be on the name...

Reply:
I don't think we need unnamed channels. The reason I used the name `NamedChannel` is just to aid in connecting this with the main text, in the end I would probably just call it `Channel` if it turns out we indeed need this class at all.